stackdriver-errors.js

```diff
@@ -30,6 +30,8 @@
   /**
    * Initialize the StackdriverErrorReporter object.
    * @param {Object} config - the init configuration.
+   * @param {Object} [config.context={}] - the context in which the error occurred.
+   * @param {string} [config.context.user] - the user who caused or was affected by the error.
```

Comment:
I wonder if this should only be in the initialization. I would have no problem if the user was never changing for a given session. But I can see use cases where the user is changing, for example a log-in flow of a webapp.

I'd suggest to create an additional method to set the user only?

Reply:
Yep ok - made sense for my case but I see your point.